packages/vscode-extension/lib/babel_transformer.js

```diff
@@ -204,6 +206,12 @@ function transformWrapper({ filename, src, ...rest }) {
     const { version } = requireFromAppDir("react-native/package.json");
     const majorMinorVersion = version.split(".").slice(0, 2).join(".");
     src = `module.exports = require("__RNIDE_lib__/rn-internals/rn-internals-${majorMinorVersion}.js");`;
+  } else if (isTransforming("/lib/polyfills/babel_transform/FetchPolyfill.js")) {
```

Comment:
why can't we scan for imports of `lib/network/interceptors/PolyfillFetchInterceptor` and then replace if with empty module when fetch-api is not found? We wouldn't need to have the placeholder module in this scenario